<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="control_widget" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>330</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>330</height>
           </size>
          </property>
          <widget class="QToolButton" name="open_button">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Open Valve</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="control_dial">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>130</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
          <widget class="QToolButton" name="close_button">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Close Valve</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="control_button">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Control</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="reset_button">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QLCDNumber" name="pressure_lcd">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>250</y>
             <width>121</width>
             <height>71</height>
            </rect>
           </property>
           <property name="digitCount">
            <number>4</number>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
          <widget class="QToolButton" name="show_button">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>210</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Show Plots</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
